feat(www): Partner Catalog update (#46757)
supabase/supabase: apps/docs/content/guides/integrations/partner-catalog.mdx

---
id: 'integrations'

> title: 'Partner Catalog'

Reviewer: Are there any other broken link issues to consider? A redirect?

Author: Yep should have all redirects in place:

https://github.com/supabase/supabase/pull/46757/changes#diff-1ab73334ee774488977594c605fe52b891b6242ffb27a612dfe13bcc0f5d2a56L2931-R2963

description: 'Integrations and Partners'
---

> The [Partner Catalog](/partners/catalog) is Supabase's public directory of third-party integrations and partners, listing tools that extend your Supabase project with external Auth, Caching, Hosting, and Low-code tools. It's distinct from [Dashboard Integrations](/guides/integrations#supabase-dashboard-integrations), which are installed directly from within a project in the Supabase Dashboard.

Reviewer: ```suggestion
The [Partner Catalog](/partners/catalog) is Supabase's public directory of third-party integrations. It lists tools that extend your Supabase project. These tools cover Auth, Caching, Hosting, and Low-code categories.

The Partner Catalog is different from [Dashboard Integrations](/guides/integrations#supabase-dashboard-integrations). You install Dashboard Integrations directly from a project in the Supabase Dashboard.
```
Cleaned up for simplicity and clarity.
